docs/resources/lakeview_dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "Workspace"
+---
+# databricks_lakeview_dashboard Resource
+
+This resource allows you to manage Databricks [Lakeview Dashboards](https://docs.databricks.com/api/workspace/lakeview). To manage [Lakeview Dashboards](https://docs.databricks.com/api/workspace/lakeview) you must have a warehouse access on your databricks workspace.
+
+## Example Usage
+
+Dashboard using `serialized_dashboard` attribute:
+
+```hcl
+resource "databricks_lakeview_dashboard" "dashboard" {
+			display_name			= 	"New Dashboard"
+			warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			serialized_dashboard	        =	"{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page\"}]}"
+			embed_credentials		=	false // Optional
+			parent_path			= 	"/Shared/provider-test"
+		}
+```
+
+Dashboard using `file_path` attribute:
+
+```hcl
+resource "databricks_lakeview_dashboard" "dashboard" {
+            display_name			= 	"New Dashboard"
+            warehouse_id			=	"{env.TEST_DEFAULT_WAREHOUSE_ID}"
+            file_path				=	"${path.module}/dashboard.json"
+            embed_credentials   		=	false // Optional
+            parent_path				= 	"/Shared/provider-test"
+        }
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `display_name` - (Required) The display name of the Lakeview dashboard.
+* `warehouse_id` - (Required) The warehouse ID used to run the dashboard.
+* `serialized_dashboard` - (Optional) The contents of the dashboard in serialized string form. Conflicts with `file_path`.
+* `file_path` - (Optional) The path to the dashboard JSON file. Conflicts with `serialized_dashboard`.
+* `embed_credentials` - (Optional) Whether to embed credentials in the dashboard. Default is `true`.
```

Comment:
Should we go ahead with default embed credentials?

Reply:
It is written in the API specs that default is true
https://docs.databricks.com/api/workspace/lakeview/publish